index.js

```diff
@@ -1,10 +1,9 @@
 
 export { fetchingData } from './src/artemisa/component'
-import { artemisa } from './src/artemisa/reducer'
-import { dataService } from './src/core/service'
+export { artemisa as reducer, isFetching, isFetched, processApiAction } from './src/artemisa/reducer'
```

Comment:
Yes, for sure we must think on how and what to export.
I've been exporting "all that I needed for some other project" but it should be "designed".
And yes, I also suspected that "reducer" is a good name from artemisa point of view, but could collide.
artemisaReducer sounds good.
Otherwise one could do

```javascript
import artemisa from 'artemisa'

artemisa.reducer
```